### DISCUSSION TITLE
Jitter is unstable when open "-F" on the server

### DISCUSSION THREAD

**Discussion by cuiminghao8** [jamulussoftware/jamulus#3043] [Technical Support & Help]
Server info: 

- Ali cloud 2core(vCPU) 8 GiB RAM
- OS: Windows Server 2022 
- Bandwidth: 100Mb

The jitter is unstable when opening "- F" option on the server, but is stable when turning it off

**Comment by dcorson-ticino-com** [comment]
This is a question of the network quality and response. 
As network quality is not a given it is possible to turn off small buffers by not using -f.
There is little or nothing that you can do as it depends on the providers.
